x/authority/types/authorization_list_test.go

```diff
@@ -399,6 +399,7 @@ func TestDefaultAuthorizationsList(t *testing.T) {
 			sdk.MsgTypeURL(&observertypes.MsgUpdateChainParams{}),
 			sdk.MsgTypeURL(&observertypes.MsgEnableCCTX{}),
 			sdk.MsgTypeURL(&observertypes.MsgUpdateGasPriceIncreaseFlags{}),
+			sdk.MsgTypeURL(&observertypes.MsgUpdateOperationalFlags{}),
```

Comment:
We would also need to update the authorization list saved to the state via `AddAuthorisation`.
Can you add an issue to track that

Reply:
I think we could do that via a migration right?

Reply:
I don't think we would need a migration script, it might just be easier to add it via message, wdyt ?

Reply:
Yes, better to add it manually and avoid migration script, we can open a operation issue in protocol-private

Reply:
Migration scripts are meant for state migrations of incompatible data between two releases and not for tasks like this. 
If you could create an issue for any problem that occurred when adding authorizations, I would be happy to look into it

Reply:
If an upgrade will break an authorization that already exists, then we should use migration scripts to ensure continuity of service and that everything continues to work as it did prior to the upgrade. 

If the authorization is new then it should be added on-chain manually instead of changing authorizations unnecessarily via migration scripts. 

I am not 100% sure `AddAuthorisation` is going to be doing. But it looks like a brand new message type right? I think we should set up the permissions manually using messages after the upgrade goes live.

Reply:
>I am not 100% sure AddAuthorisation is going to be doing. But it looks like a brand new message type right? I think we should set up the permissions manually using messages after the upgrade goes live.

No we've been using it multiple time.

The upgrade doesn't break anything, the authorization can be added anytime before the first use of the message.

>I definitely disagree since we seem to always mess up these manual authorization migrations. We really should aim to do as few manual steps as possible IMO.

We did once for a message that was needed right at the upgrade (VoteGasPrice) this would have required a migration script to be functional as soon as the upgrade happen. We don't have this requirement here. We just need to track an issue to add it.

Reply:
https://github.com/zeta-chain/protocol-private/issues/204